packages/usa-radio/src/usa-radio.stories.js

```diff
@@ -9,6 +9,11 @@ export default {
       control: { type: "radio" },
       options: ["none", "disabled", "aria-disabled"],
     },
+    indeterminate: {
+      name: "Indeterminate option (toggle to reset)",
+      control: { type: "boolean" },
+      defaultValue: false
+    }
```

Comment:
## New storybook control

Added new storybook control to radio so it's easy to test `indeterminate` or "unselected` radio styles.

Toggle the control to remove selection!